catalogue/webapp/pages/works/[workId]/index.tsx

```diff
@@ -66,7 +66,7 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
           // we shouldn't overload this
           // these metrics allow us to report back on breadth of collection accessed
           properties: {
-            workType: workResponse.workType.id,
+            workType: workResponse.workType?.id,
```

Comment:
@harrisonpim @taceybadgerbrook i don't know if this'll cause issues but FYI the ID of a work might be `undefined` now in pageview properties (tbh it was probably already doing that, but this makes it official)